pom.xml

```diff
@@ -147,6 +148,16 @@
       <groupId>org.springframework.boot</groupId>
       <artifactId>spring-boot-starter-test</artifactId>
       <scope>test</scope>
+      <exclusions>
+        <exclusion>
+          <groupId>junit</groupId>
+          <artifactId>junit</artifactId>
+        </exclusion>
+        <exclusion>
+          <groupId>org.junit.vintage</groupId>
+          <artifactId>junit-vintage-engine</artifactId>
+        </exclusion>
+      </exclusions>
```

Comment:
Do you exclude JUnit 4 and the associated test engine for only being dependent on JUnit 5's Jupiter?

Is this the canoncial way of configuring a Spring Boot test scope?

Reply:
Removed the exclusion.